gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/ConfigStoreUtils.java

```diff
@@ -260,8 +262,11 @@ public static void setTopicsFromConfigStore(Properties properties, Set<String> b
   }
 
   public static List<String> getListOfValuesFromConfigStore(Config config, String keyValue) {
```

Comment:
Doesn't ConfigUtils#getStringList() achieve the same? If so, should we reuse the method defined there?

Reply:
That's a good catch, will address.